src/build.py

```diff
@@ -592,6 +596,7 @@ def Binaryen():
     f = os.path.join(BINARYEN_BIN_DIR, node)
     if os.path.isfile(f):
       CopyBinaryToArchive(f)
+  CopyBinaryToArchive(os.path.join(BINARYEN_SRC_DIR, 'bin', 'wasm.js'))
```

Comment:
Yup. we could rename one or both of them in their respective repos, but for now waterfall just needs to put them where stuff expects them.